python/cudf/tests/test_dataframe.py

```diff
@@ -1954,7 +1954,7 @@ def test_dataframe_empty_sort_index():
     assert_eq(expect, got)
 
 
-@pytest.mark.parametrize('dtype', ['int8', 'int16', 'int32', 'int64',
+@pytest.mark.parametrize('dtype', ['bool', 'int8', 'int16', 'int32', 'int64',
                                    'float32', 'float64', 'datetime64[ms]',
                                    'category'])
 def test_dataframe_0_row_dtype(dtype):
```

Comment:
Maybe we need a new test to validate the new functionality? It's not just bool, you're fixing all iloc dtypes here, but that isn't reflected in the test config. No tests were failing either, right?

I'm guessing that tests were written for Iloc value returns, but not for Iloc types.